torch/_torch_docs.py

```diff
@@ -2653,24 +2653,23 @@ def merge_dicts(*dicts):
            r"""
 lu_solve(b, LU_data, LU_pivots, out=None) -> Tensor
 
-Batch LU solve.
-
 Returns the LU solve of the linear system :math:`Ax = b` using the partially pivoted
 LU factorization of A from :meth:`torch.lu`.
 
 Arguments:
-    b (Tensor): the RHS tensor
-    LU_data (Tensor): the pivoted LU factorization of A from :meth:`torch.lu`.
-    LU_pivots (IntTensor): the pivots of the LU factorization
+    b (Tensor): the RHS tensor of size :math:`(b, m, *)`, where :math:`*` is a zero or one
```

Comment:
Maybe we shouldn't document that `*` can be zero dimensions to discourage use of it.

Reply:
I think this seems reasonable. Will modify accordingly.